rdf/common/src/main/scala/org/w3/banana/Prefix.scala

```diff
@@ -14,7 +14,7 @@ object Prefix {
     new PrefixBuilder(prefixName, prefixIri)
 }
 
-private [banana] class PrefixBuilder[Rdf <: RDF](
+class PrefixBuilder[Rdf <: RDF](
```

Comment:
The guy you made that private must have been drunk when he wrote that code :-)  +1 to make it public. And it won't break existing code.